frontend/src/libs/api.js

```diff
@@ -20,6 +20,15 @@ export const createTodo = (todosType, cardInfo) => {
   });
 };
 
+export const getNotifications = () => {
+  return fetch(`${BASE_URL}/noti`, {
+    method: 'GET',
+    headers: {
+      'Contetn-Type': 'application/json',
```

Comment:
```suggestion
      'Content-Type': 'application/json',
```

오타가 있어요!

body가 없어서 문제 없기는 하지만요!